pkg/core/interop_neo.go

```diff
@@ -90,6 +98,11 @@ func (s *storageIterator) Value() stackitem.Item {
 			panic(err)
 		}
 	}
+	if s.opts&FindPick0 != 0 {
+		value = value.Value().([]stackitem.Item)[0]
```

Comment:
Can we add a test for non-array and zero-length array cases? They should panic of course.

Reply:
Fixed.